smarts/core/remote_agent_buffer.py

```diff
@@ -32,17 +32,23 @@
 
 
 class RemoteAgentBuffer:
-    def __init__(self, zoo_worker_addrs=None, buffer_size=3):
+    def __init__(self, zoo_worker_addrs=None, authkey=b"secret", buffer_size=3):
```

Comment:
```suggestion
    def __init__(self, zoo_worker_addrs=None, auth_key=None, buffer_size=3):
```

Reply:
Changed default value of auth_key of type byte string to be b"None".